libraries/AP_Motors/AP_MotorsHeli_Quad.cpp

```diff
@@ -177,7 +177,10 @@ uint16_t AP_MotorsHeli_Quad::get_motor_mask()
     for (uint8_t i=0; i<AP_MOTORS_HELI_QUAD_NUM_MOTORS; i++) {
         mask |= 1U << (AP_MOTORS_MOT_1+i);
     }
-    mask |= 1U << AP_MOTORS_HELI_QUAD_RSC;
+    uint8_t rsc_chan;
+    if (SRV_Channels::find_channel(SRV_Channel::k_heli_rsc, rsc_chan)) {
```

Comment:
In my opinion it would be better to do something like `_rotor.get_output_mask()` (which should use the SRV_Channel `get_output_channel_mask` method). This could be used in several other places.

Reply:
Thanks... but I still don't understand why I needed to change this to use a non-default RSC channel on px4-v4 (channel 5 instead of 8). Just setting parameter servo5_function to k_heli_rsc works in SITL, but I wasn't able to spin the motor on the real heli-quad without this change. This method doesn't even seem to affect the code path used by _rotor.write_rsc